### ISSUE TITLE
Mono variable or condensed

### ISSUE THREAD

**Issue by devinhalladay** [IBM/plex#462]
Hi! Thanks to the whole Plex team for their work on this project :)

Are there any plans to add either a variable cut of Mono, or a Mono Condensed variant?

Thanks!

**Comment by BoldMonday** [comment]
A variable font version of Plex Mono is on the road map although other things have more priority at the moment.
Plex Mono Condensed has not been discussed so far.

**Comment by devinhalladay** [comment]
Thanks for the context @BoldMonday, I'll stay tuned then :)

**Comment by jameshfisher** [comment]
@BoldMonday since this is on the roadmap, could we keep this issue open for visibility? I'm also very interested in a Mono Condensed. We're using Plex on https://tigyog.app/, but in code blocks, we have to keep to < 40 chars per line due to the width of the mono font. This is very restrictive.

I'd also be happy with a recommendation for an alternative mono condensed font that works well with the Plex family.

**Comment by JimEBlevins** [comment]
> [….] I'm also very interested in a Mono Condensed. We're using Plex on https://tigyog.app/, but, in code blocks, we have to keep to < 40 chars per line due to the width of the mono font. [….]
> 
> I'd also be happy with a recommendation for an alternative mono condensed font that works well with the Plex family.

The monospaced font **Iosevka** is relatively condensed, and it has a _stylistic set_ [in the spirit of](https://github.com/be5invis/Iosevka/issues/1224#issuecomment-930750342) **IBM  Plex Mono** ([Iosevka ss15](https://github.com/be5invis/Iosevka/blob/main/doc/stylistic-sets.md)): ([svg file of a _light_ weight](https://github.com/be5invis/Iosevka/blob/main/images/ss-u-ss15-1.light.svg#gh-light-mode-only))

Iosevka also has an _expanded_ variant, which is packaged with each of its [styles (fixed and with coding ligatures) in many formats](https://github.com/be5invis/Iosevka/blob/main/doc/PACKAGE-LIST.md).

**Comment by uni-0000** [comment]
A condensed version of Plex Mono is a great idea, I would love something close to Letter Gothic.

**Comment by BoldMonday** [comment]
To summarise and clarify the current situation:
A variable version of Plex Mono is on the roadmap, although fixing bugs still has a higher priority.
A condensed version of Plex Mono is not on the roadmap.

**Comment by uni-0000** [comment]
> To summarise and clarify the current situation: A variable version of Plex Mono is on the roadmap, although fixing bugs still has a higher priority. A condensed version of Plex Mono is not on the roadmap.

The variable version support WDTH?

**Comment by BoldMonday** [comment]
> The variable version support WDTH?

No. Only a WGHT axis in the same weight range as the current static fonts.
A WDTH axis would require the design of a Condensed version.

**Comment by fidaay** [comment]
Hello, are there updates on this issue?

**Comment by BoldMonday** [comment]
Plex Mono Var release: https://github.com/IBM/plex/releases/tag/%40ibm%2Fplex-mono-variable%401.0.0
